src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -801,15 +802,15 @@ private void cleanupConnection(TaskListener listener) {
      * return javaPath if specified in the configuration.
      * Finds local Java, and if none exist, install one.
      */
-    protected String resolveJava(SlaveComputer computer, TaskListener listener) throws InterruptedException, IOException2 {
+    protected String resolveJava(SlaveComputer computer, TaskListener listener) throws InterruptedException, IOException {
```

Comment:
Does not seem to be worth the hassle. Only clients impacted are those that handle `IOException2` and `IOException` differently - which does not really makes sense. Fortunately, this will break source compatibility pointing them to the problem.